sentences/fr-CA/_common.yaml

```diff
@@ -49,18 +49,18 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
-  name: "([le | la | les] {name} | [l']{name})"
-  area: "([le | la ] {area} | [l']{area})"
-  brightness: "{brightness:brightness}[%| pourcent]"
-  temperature: "{temperature}[°| degrés] [{temperature_unit}]"
-  regle: "(règle | met | ajuste | change)"
-  allume: "(allume | active | démarre | ouvre)"
-  eteins: "(éteint | éteins | désactive | stoppe | arrête | coupe | ferme )"
-  lumiere: "(la lumière | la lampe | l'ampoule)"
-  lumieres: "[les] (lumières | lampes | ampoules)"
-  ventilateur: "(le ventilateur | l'échangeur d'air | la fan)"
-  ventilateurs: "(les ventilateurs | les fans)"
-  dans: "(dans | du | de)"
+  name: "([le|la|les] {name} | [l']{name})"
+  area: "([le|la] {area}|[l']{area})"
+  brightness: "{brightness:brightness}[%|pourcent]"
```

Comment:
there must be a space between `[%| percent]` otherwise brightness will stick together with percent with no space.